velox/dwio/parquet/tests/reader/CMakeLists.txt

```diff
@@ -12,7 +12,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+add_custom_target(example_file_dir ALL)
```

Comment:
I get what you are doing here, but I reckon this must be a common enough pattern that there must be a simpler , idiomatic way of doing this. 

cc : @assignUser